find-cursor.c

```diff
@@ -59,6 +60,8 @@ void usage(char *name) {
 	printf("                      some display issues when following the cursor position,\n");
 	printf("                      but it doesn't work well with all WMs, which is why\n");
 	printf("                      it's disabled by default.\n");
+	printf("  -i, --inverted      Draw the inverted of the given color additionally\n");
+	printf("                      to see the circles on all backgrounds\n");
```

Comment:
Minor issue, but you will have to add the `-i` flag to the `printf("Usage: ...")` on line 45 too.

Reply:
Also, I wonder if this can't be rephrased in a better way. What it does now is draw a border around every one of the pixels, so maybe `-b` and `--border` makes more sense? Or perhaps `--outline`? Or something else?

Right now I had to look at the actual implementation to see what it does, I thought it wasn't very obvious from just the help description.